examples/python/geometry/point_cloud_bounding_box.py

```diff
@@ -25,14 +25,10 @@
 # ----------------------------------------------------------------------------
 
 import open3d as o3d
-import os, sys
-
-pyexample_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-test_data_path = os.path.join(os.path.dirname(pyexample_path), 'test_data')
-sys.path.append(pyexample_path)
 
 if __name__ == "__main__":
-    pcd = o3d.io.read_point_cloud(os.path.join(test_data_path, 'fragment.ply'))
+    sample_ply_data = o3d.data.DemoCropPointCloud()
```

Comment:
use `SamplePointCloudPLY`, if you just want to use `fragment.ply`, and not `cropped.json` in an example.

Reply:
Thanks for pointing out, didn't know this existed.